back/app/services/custom_field_service.rb

```diff
@@ -139,20 +139,22 @@ def self.can_see_admin_answers?(idea, current_user)
     idea.author_id == current_user.id || UserRoleService.new.can_moderate_project?(idea.project, current_user)
   end
 
-  private
```

Comment:
Basically just made these existing methods public so I can use them from the PDF and XLS exporters